README.md

```diff
@@ -120,49 +120,49 @@ However, often we don't even realize that the code being tested is using the net
+Unit tests should ideally not rely on the current filesystem to run successfully. If they do, they are slower as well as more error prone as the filesystem can change.
 
-One alternative is [mocking disk i/o calls](https://nickolaskraus.io/articles/how-to-mock-the-built-in-function-open/).
+One alternative is [mocking disk I/O calls](https://nickolaskraus.io/articles/how-to-mock-the-built-in-function-open/).
 
 Another is a temporary in-memory filesystem, provided by [`pyfakefs`](https://github.com/pytest-dev/pyfakefs), making your tests both safer and faster.
 
 ## Database access
 
-Testing web apps usually requires some database access in tests, and that's OK. There are still several optimizations possible.
+Testing web apps usually requires some database access in tests, and that's OK. There are still several optimizations available.
 
 ### Do all tests need a database?
 
-Some tests could potentially be rewritten in a way that does not require a database to do the testing. For example, if calculating a user's age, there is no need to fetch a birth date from a DB. It's better to provide a fake birth date to the calculation function, and remove the test's dependency on the database fixture.
+Some tests could potentially be rewritten in a way that does not require a database to do the testing. For example, if calculating a user's age, there is no need to fetch a birth date from a DB. It's better to provide a fake birth date to the calculation function and remove the test's dependency on the database fixture.
 
 ### Do all tests need the entire database?
 
-Populating the database with test data takes time. Do all tests need the entire dummy dataset? If a group of test is testing some user profile feature, they potentially don't need to populate non-relevant tables in the database.
+Populating the database with test data takes time. Do all tests need the entire dummy dataset? If a group of tests is testing some user profile feature, they potentially don't need to populate non-relevant tables in the database.
 
 ### Only prepare the database once
 
-Preparing, using and then destroying the dummy database for every test is wasteful. There are better approaches.
+Preparing, using, and then destroying the dummy database for every test is wasteful. There are better approaches.
 
 #### Truncate
 
-Instead of destroying the database after a test run, rather [`TRUNCATE` its tables](https://www.niklas-meinzer.de/post/2019-07_pytest-performance/#database-setup-and-tear-down). This saves your from re-creating the database for the next test.
+Instead of destroying the database after a test run, rather [`TRUNCATE` it's tables](https://www.niklas-meinzer.de/post/2019-07_pytest-performance/#database-setup-and-tear-down). This saves you from recreating the database for every test that relies on it.
```

Comment:
Are you sure about `its` -> `it's`?

Reply:
I'm not 100% sure, we can keep it at `its`.